oe_corporate_site_info.info.yml

```diff
@@ -7,3 +7,5 @@ core: 8.x
 
 dependencies:
   - rdf_skos:rdf_skos
+  # Dependency is deprecated and has to be removed after 2.0.0 release of oe_content. Required for configuring needed RDF SKOS graphs.
```

Comment:
I don't think we need this dependency at all. This component only requires one graph  (http://publications.europa.eu/resource/authority/corporate-body) and we can configure it on an install method by calling \Drupal::service('rdf_skos.skos_graph_configurator')->addGraphs(GRAPH);